exercises/G-conditionals-2/exercise-1.js

```diff
@@ -15,6 +15,14 @@ let number1 = 5;
 let number2 = -1;
 let number3 = 0;
 
+function negativeOrPositive (number){
```

Comment:
This is correct. Duplicate function again though (the `function negativeOrPositive(number) {}` line should be deleted)